README.md

```diff
@@ -109,6 +109,8 @@ var startupPromise = runner.start({
    you must override the app base path with either:
      - `APP_BASE_PATH` environment variable
      - `app_base_path` config stanza.
+- If project requires cancellable promises (which is disabled by default) you must set the
```

Comment:
Could you add a link to the documentation regarding cancellable promises here?

Reply:
`If the project requires cancellable promises (which are disabled by default)`